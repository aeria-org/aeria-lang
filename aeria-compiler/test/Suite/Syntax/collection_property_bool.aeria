collection CollectionPropertyBoolTest {
  properties {
    prop bool
  }
}
