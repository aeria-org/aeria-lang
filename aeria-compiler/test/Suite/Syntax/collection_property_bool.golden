{
  "kind": "Program",
  "collections": [
    {
      "properties": [
        {
          "type": {
            "kind": "PBoolean"
          },
          "name": {
            "value": "prop",
            "kind": "PropertyName"
          },
          "kind": "Property"
        }
      ],
      "name": {
        "value": "collectionpropertybooltest",
        "kind": "CollectionName"
      },
      "kind": "Collection"
    }
  ]
}
